shell/pom.xml

```diff
@@ -67,6 +67,10 @@
       <groupId>org.apache.commons</groupId>
       <artifactId>commons-compress</artifactId>
     </dependency>
+    <dependency>
```

Comment:
@LuQQiu as mentioned today by Baolong, this only adds dependency to shell module so no dependency added to the client

Reply:
Yeah.